README.md

```diff
@@ -56,7 +56,8 @@ Mesos, designed for high performance data processing jobs that require flexibili
 ### Streaming Library
 
 - [Apache Kafka Streams](https://github.com/apache/kafka) [Java] - lightweight stream processing library included in Apache Kafka (since 0.10 version).
-- [Akka Streams](https://github.com/akka/akka) [Scala] - stream processing library on Akka Actors. 
+- [Akka Streams](https://github.com/akka/akka) [Scala] - stream processing library on Akka Actors.
+- [Daggy](https://github.com/synacker/daggy) [C++] - real-time streams aggregation and catching. 
```

Comment:
please add your library to just one category.

Reply:
The project has [library](https://docs.daggy.dev/daggy-core) and [application](https://docs.daggy.dev/daggy-console-application). Can I post them in separate categories?

Reply:
Most libraries have applications as well. Just add it as library will do.

Reply:
Removed daggy from applications.